Formula/ponysay.rb

```diff
@@ -12,6 +13,8 @@ class Ponysay < Formula
     sha256 "a502ed3340bc2c7591788c8747c8175e0ac7902bfbfdf73454aef09d39a0db16" => :high_sierra
   end
 
+  depends_on "gzip" => :build
+  depends_on "texinfo" => :build
```

Comment:
```suggestion
  uses_from_macos "texinfo" => :build
```

Reply:
Also separate `depends_on` and `uses_from_macos` with a newline.

Reply:
I'm sorry, what is `uses_from_macos`? I miss somewhere didn't see that on docs. btw I use linux.

Reply:
`uses_from_macos` is a stanza used for software that is included in macOS. It is a no-op on macOS and a normal dependency on Linux. `texinfo` is provided by macOS.

Reply:
Okay thank you. Probably I just missed it. I'll do it later when I back, I'm outside atm.